crawler/src/main/java/com/brandwatch/minibcr/crawler/config/RedditConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.brandwatch.minibcr.crawler.config;
+
+import com.brandwatch.minibcr.crawler.service.reddit.auth.RedditClient;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class RedditConfig {
+    static private final String CLIENT_ID = "Fll4jWWRYjJKYYZUHu6rSQ";
+    static private final String CLIENT_SECRET = "1A0gdRtSkCI31qumAMIRRIeSCql2tw";
```

Comment:
These really should be env variables that are injected, rather than stored in a variable. 

You will probably need to regenerate these now, as bots often scrape github looking for secrets, and might send requests with your credentials.

Reply:
This one is really important!